pages/tutorials/graphql_sso.md.erb

```diff
@@ -0,0 +1,250 @@
+# Manual SSO Setup with GraphQL
+
+Buildkite’s SSO login can be set up by emailing support, or you can set it up manually using our [GraphQL APIs](/docs/apis/graphql). This tutorial covers how to set up SSO manually using GraphQL, for most of the common SSO providers.
+
+For details on every available option available in the GraphQL APIs, please use the documentation sidebar built into the [GraphQL Explorer](/docs/graphql-api#getting-started).
+
+<%= toc %>
+
+## Finding your organization’s GraphQL ID
+
+For every type of SSO provider, you’ll need your Buildkite organization’s GraphQL ID. You can find the organization `id` using the following GraphQL query:
```

Comment:
In all the other examples, we've got the name of the query, what do you think about adding that here rather than 'the following'?

Reply:
Ah, the other examples show the mutation/field names. This one is a bit different, because `OrganizationId` isn't anything in our API at all… it's just the label we've given the query:

```graphql
query OrganizationId {
  organization(slug: "myorg") {
    id
  }
}
```

It's actually optional, and I'm tempted to remove it:

```graphql
query {
  organization(slug: "myorg") {
    id
  }
}
```

but then, if you copy + paste it into the GraphQL Console/Explorer, you can't choose between the multiple queries/mutations if they don't have a label. Argh!

Reply:
I tweaked the words to try to make it clearer…